plugins/linter-eslint.yml

```
@@ -7,5 +7,5 @@ icon: linter-eslint.png
 github: https://github.com/amplication/plugins/tree/master/plugins/linter-eslint
 website: https://github.com/amplication/plugins/tree/master/plugins/linter-eslint
 type: official
-categories: [linter]
+categories: [Code styling, Static code analysis]
```

is this valid yaml? Do we need to make them strings?
i.e.
 ```suggestion
categories: ['Code styling', 'Static code analysis']
```

---

Nevermind, it's a valid YAML as per https://yaml.org/spec/1.2.2/#sequence